transaction/main.go

```diff
@@ -260,6 +260,11 @@ func getStockPortfolio(c *gin.Context) {
 		portfolio = append(portfolio, item)
 	}
 
+	// Check if the portfolio is empty and set to nil if so
+	if len(portfolio) == 0 {
+		portfolio = nil
```

Comment:
don't we want this to be an empty list so that its
`"success":true,"data":[]}` and not `"success":true,"data":null}`?